content/includes/stories.yaml

```diff
@@ -0,0 +1,161 @@
+      image_width: 116
+      image_height: 35
+      class: time
+web_stories:
+  title@: Check out AMP stories on the web
+  stories:
+    - base_image: /static/img/stories/story-cnn
+      image_width: 385
+      image_height: 641
+      immage_alt@: The Missions Still Exploring Our Solar System
+      base_logo: /static/img/stories/logo-cnn
+      logo_width: 395
+      logo_height: 45
+      logo_alt@: CNN
+      url: https://stamps.cnn.io/solar-system-missions/index.html
+    - base_image: /static/img/stories/story-hearst
+      image_width: 385
+      image_height: 641
+      immage_alt@: Apple Cider Vinegar is the Magic Potion You Need in Your Life
+      base_logo: /static/img/stories/logo-hearst
+      logo_width: 395
+      logo_height: 45
+      logo_alt@: Hearst
+      url: http://www.cosmopolitan.com/stamp/apple-cider-vinegar/
+    - base_image: /static/img/stories/story-mashable
+      image_width: 385
+      image_height: 641
+      immage_alt@: Black Panther
+      base_logo: /static/img/stories/logo-mashable
+      logo_width: 395
+      logo_height: 45
+      logo_alt@: Mashable
+      url: https://mashable.com/2017/10/24/reels-guide-to-black-panther.amp
+    - base_image: /static/img/stories/story-time
+      image_width: 385
+      image_height: 641
+      immage_alt@: Harry & Meghan A to Z
+      base_logo: /static/img/stories/logo-meredith
+      logo_width: 395
+      logo_height: 45
+      logo_alt@: Meredith
+      url: https://people.com/amp-stories/royal-a-to-z/
+    - base_image: /static/img/stories/story-mic
+      image_width: 385
+      image_height: 641
+      immage_alt@: The Art of an Organized Refrigerator
+      base_logo: /static/img/stories/logo-mic
+      logo_width: 395
+      logo_height: 45
+      logo_alt@: Mic
+      url: https://mic.com/stories/262/the-art-of-an-organized-refrigerator
+    - base_image: /static/img/stories/story-vox
+      image_width: 385
+      image_height: 641
+      image_alt@: How College Football Playoff Rankings Actualloy Work
+      base_logo: /static/img/stories/logo-vox
+      logo_width: 395
+      logo_height: 45
+      logo_alt@: Vox
+      url: https://www.sbnation.com/2017/12/1/16719476/college-football-playoff-rankings-explained
+    - base_image: /static/img/stories/story-wapo
+      image_width: 385
+      image_height: 641
+      immage_alt@: The Evolution of Trump's Wall
+      base_logo: /static/img/stories/logo-wapo
+      logo_width: 395
+      logo_height: 45
+      logo_alt@: Washington Post
+      url: https://www.washingtonpost.com/graphics/2018/national/border-wall/
+    - base_image: /static/img/stories/story-wired
+      image_width: 385
+      image_height: 641
+      immage_alt@: Space Photos of the Week
+      base_logo: /static/img/stories/logo-wired
+      logo_width: 395
+      logo_height: 45
+      logo_alt@: Wired
+      url: http://stamp.wired.com/stamp/space-photos-of-the-week-111817?key=wired_stamp_m3aIz8a21lyTY11n
```

Comment:
Please change  Wired link to: https://www.wired.com/amp-stories/space-photos-of-the-week-111817/